portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,46 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+// function addRandomGreeting() {
+//   const greetings =
+//       ["What we know is a drop and what we don't know is an ocean.", "Beets, Bears, Battlestar Galactica", "To live is to risk it all.", "People don't change. They become more of who they are."];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+//   // Pick a random greeting.
+//   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+
+//   // Add it to the page.
+//   console.log("this is a function for random quotes");
+//   const greetingContainer = document.getElementById('greeting-container');
+//   greetingContainer.innerText = greeting;
+// }
+
+function addProjectDetail(buttonId) {
+  const description =
+      [
+       "A Machine Learning Bot classifier used for segregation of Bots and Humans on Twitter during Indian socio-political uproar.",
+       "A Machine Learning image classifier to compare accuracy of KNN (with SVM) model and Neural Networks model for character recognition.", 
+       "A portal for organizing information and making appointments with healthcare professionals.", 
+       "A Google Maps & Weather API based application for real-time weather info on Maps with additional functionalities."
+      ];
+
+  // Add description to the page. 
+  const descriptionDiv = document.getElementById(buttonId+"-div");
+
+  switch(buttonId) {
+    case "a":
+      descriptionDiv.innerText = description[0];
```

Comment:
Can you use some setter methods instead of directly modifying the property?

```suggestion
      descriptionDiv.setText(description[0]);
```

Reply:
@sonudoo I made the other changes you recommended. 
One change I am unable to add is a custom setter for DOM element property as it isn't recommended to override set property. Will it be possible to suggest an alternative for that?